.github/workflows/manage-issue-header.yml

```diff
@@ -1,47 +1,46 @@
 name: Manage issue header
 
 on:
-    issues:
-      types: [labeled, unlabeled]
-    workflow_call:
-      secrets:
-        LE_BOT_APP_ID:
-            description: "GitHub App ID for authentication"
-            required: true
-        LE_BOT_PRIVATE_KEY:
-            description: "GitHub App Private Key for authentication"
-            required: true
+  workflow_call:
+    secrets:
+      LE_BOT_APP_ID:
+          description: "GitHub App ID for authentication"
+          required: true
+      LE_BOT_PRIVATE_KEY:
+          description: "GitHub App Private Key for authentication"
+          required: true
+
 jobs:
   manage-issue-header:
     runs-on: ubuntu-latest
     if: |
+      github.event.action == 'opened' ||
```

Comment:
Would need to add the other event names here too, presumably.

Reply:
Yes, thanks!